src/main/java/org/eolang/jetbrains/EoCompileAction.java

```diff
@@ -0,0 +1,110 @@
+import org.jetbrains.idea.maven.execution.MavenRunConfigurationType;
+import org.jetbrains.idea.maven.execution.MavenRunnerParameters;
+import org.jetbrains.idea.maven.model.MavenExplicitProfiles;
+import org.jetbrains.idea.maven.model.MavenPlugin;
+import org.jetbrains.idea.maven.project.MavenProject;
+import org.jetbrains.idea.maven.project.MavenProjectsManager;
+import org.jetbrains.idea.maven.utils.actions.MavenActionUtil;
+
+/**
+ * Compile EO sources.
+ * @since 1.0
+ * @todo #20:30min Make compile action complete adding remaining compile actions
+ *  (assemble, transpile etc.)
+ * @todo #20:30min Create tests for the action
+ */
+public final class EoCompileAction extends AnAction {
+    @Override
+    public void actionPerformed(@NotNull final AnActionEvent action) {
+        final DataContext context = action.getDataContext();
+        final Project project = MavenActionUtil.getProject(context);
+        MavenProject maven = null;
+        String name = "";
+        if (project != null) {
+            maven = MavenActionUtil.getMavenProject(context);
+            name = project.getName();
+        }
+        if (maven == null) {
+            Notifications.Bus.notify(
+                new Notification(
+                    Notifications.SYSTEM_MESSAGES_GROUP_ID,
+                    "Cannot compile EO",
+                    String.format("Not a maven project: %s", name),
+                    NotificationType.WARNING
+                )
+            );
+            return;
+        }
+        final MavenPlugin plugin = EoCompileAction.eoPlugin(maven);
+        final MavenProjectsManager manager = MavenActionUtil.getProjectsManager(context);
+        if (manager != null && plugin != null) {
+            final String version = plugin.getVersion();
+            final MavenExplicitProfiles profiles = manager.getExplicitProfiles();
+            final MavenRunnerParameters params = new MavenRunnerParameters(
+                true,
+                maven.getDirectory(),
+                maven.getFile().getName(),
+                Collections.singletonList(
+                    String.format("org.eolang:eo-maven-plugin:%s:register", version)
+                ),
+                profiles.getEnabledProfiles(),
+                profiles.getDisabledProfiles()
+            );
+            MavenRunConfigurationType.runConfiguration(project, params, null);
+        }
+    }
+
+    /**
+     * Try to find EO Maven plugin.
+     * @param maven Maven project
+     * @return Found plugin or null
+     */
+    private static MavenPlugin eoPlugin(final MavenProject maven) {
+        final MavenPlugin plugin = maven.findPlugin("org.eolang", "eo-maven-plugin");
+        if (plugin == null) {
+            Notifications.Bus.notify(
+                new Notification(
+                    Notifications.SYSTEM_MESSAGES_GROUP_ID,
+                    "Cannot compile EO",
+                    "Eo-maven plugin is not configured",
```

Comment:
@mximp pretty strange message. Let's do it this way:

```
eo-maven-plugin is not configured in your pom.xml, see https://github.com/objectionary/eo/tree/master/eo-maven-plugin
```

Reply:
fixed